docker-compose.yml

```
@@ -33,6 +33,28 @@ services:
       ELASTICSEARCH_URL: "http://hello-ltr-elastic:9200"
       SERVER_HOST: "0.0.0.0"
 
+  opensearch:
+    build:
+      context: ./notebooks/opensearch/.docker/opensearch-docker/
+      dockerfile: Dockerfile
+    container_name: hello-ltr-opensearch
+    ports:
+      - 9201:9201
+
```

I think we also need to disable security here, i.e. something like
```suggestion
    
    environment:
      - "OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m"
      - "DISABLE_INSTALL_DEMO_CONFIG=true"
      - "DISABLE_SECURITY_PLUGIN=true"
      - "discovery.type=single-node"
```

---

done